### PR TITLE
Python tutorial, first lesson.

### PR DESCRIPTION
**Pull request by katiam**

### CODE REVIEW

**Comment by despo** [comment]
@katiam thank you Katia, this is an awesome start. There are a couple of small grammatical errors, it would be great if you could correct them so I can merge this. I'll comment on the relevant lines.

File: general/setup/tutorial.md [hunk 1]
```diff
@@ -40,4 +40,6 @@ If you have a Mac you'll have a version of Ruby installed already. A coach can h
 You can [download and run the RubyInstaller program](http://rubyinstaller.org/) to program Ruby on Windows.
 
 
+## Python (optional)
 
+Python would usually set up on your operating system. Your tutor will show you how to run Python code.
```
Comment (despo) [inline general/setup/tutorial.md]: `Python usually comes with your operating system. Your coach will show you how to use it to run execute Python code.`

File: python/lesson1/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#Introduction to Python
+
+There is a very good introduction article in [Google Developers Guide](https://developers.google.com/edu/python/introduction)
+
+Also there are resources for beginners on [the Python website](https://www.python.org/about/gettingstarted/). And you can always refer to [the documentation](https://docs.python.org/2/tutorial/introduction.html) that explains the basics.
```
Comment (despo) [inline python/lesson1/tutorial.md]: You can also find resources for beginners on [the Python website](https://www.python.org/about/gettingstarted/) and refer to [the Python documentation](https://docs.python.org/2/tutorial/introduction.html), where the language basics are explained.

**Comment by katiam** [comment]
Done, thanks.

**Comment by despo** [comment]
@katiam :+1: :star2:

**Pull request merged**